default/generated/technology-usage/194-connect.windup.yaml

```diff
@@ -146,14 +162,10 @@
   description: Embedded library - AMQP Client
   labels:
   - konveyor.io/include=always
-  links:
-  - title: Red Hat JBoss Enterprise Application Platform (EAP) 6 Supported Configurations
-    url: https://access.redhat.com/articles/111663
-  - title: Red Hat JBoss Enterprise Application Platform (EAP) 7 Supported Configurations
-    url: https://access.redhat.com/articles/2026253
+  links: []
   ruleID: connect-02300
   tag:
-  - AMQP Client
+  - ' Client'
```

Comment:
Nit: Did we want the white space before Client?

Reply:
> Nit: Did we want the white space before Client?
This rule is directly copied from the upstream: https://github.com/konveyor/rulesets/blob/e1ca097b35d990cab32067b55ad1144d37d10d5c/default/generated/technology-usage/251-connect.groovy.windup.yaml#L127C1-L128C14, Let's keep it as is. If you have any further concerns, I can create a new PR to address them.